2-1-3.py

```diff
@@ -0,0 +1,12 @@
+s = input()
+sum_k = len(s) * 60
+sum_r = float(sum_k / 100)
+sum_str = str(sum_r)
+sum_str_result = sum_str
+sum_str_result = sum_str_result.replace('.', ' р. ')
```

Comment:
No need to have this assignment, you can safely delete it:
```suggestion
sum_str_result = sum_str.replace('.', ' р. ')
```